stack.go

```diff
@@ -127,6 +127,17 @@ func (s *stack) get() []StackFrame {
 	return sFrames
 }
 
+// isGlobal determines if the stack trace represents a global error
+func (s *stack) isGlobal() bool {
+	frames := s.get()
+	for _, f := range frames {
+		if f.Name == "runtime.doInit" {
```

Comment:
should we compare lowercase of each? i dont think it is necessary but mot sure if f.Name will ever be upper or camelcase

Reply:
it's safer and easy enough to change so i'll do it